### ISSUE TITLE
o-header: markup visual drawer navigation headings as headings

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#910]
Text had been visually styled as headings, without being programmatically marked up as such for users of screen reading assistive technologies.

Within the expandable menu, users are presented with text ‘Top sections’ and ‘FT recommends’ which visually introduce different sections of listed content, as they are styled as headings and also function as such. However, they have not been programmatically marked up as such as they are only `<li>`, the same as the links which they introduce. 
This means that users of screen reading assistive technologies are not provided with the same structure, information, and relationships between content as sighted users are provided with visual styling. 

**solution**
Ensure that users of screen reading assistive technologies are provided with the same semantic and programmatical structure of content, as sighted users are provided with visually. This includes instances where text visually acts and has been styled as headings and therefore, should also be marked up as a programmatic heading, preferably using a heading element.

As both the text ‘Top sections’ and ‘FT recommends’ have been visually styled as headings, they should be marked up as `<h2>` elements. Consider splitting the content up with each heading introducing their own individual lists and not be contained in one large list.

Relates to:
https://financialtimes.atlassian.net/browse/CON-2119?atlOrigin=eyJpIjoiMmE2OTE2ZGVlYjgzNDIxZjgzN2IxN2UyOGMxZGQ3NGQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ
https://docs.google.com/document/d/17-d7DAAORFCaUbiHdFPqvrBerUKbMyhG/edit#bookmark=id.6pjqvta1kbps

Similar guidelines here: https://www.w3.org/WAI/tutorials/menus/structure/#label-menus

**Comment by notlee** [comment]
fixed in `o-header@^11`
